cmd/price-feeder.go

```diff
@@ -61,6 +62,11 @@ func init() {
 	rootCmd.PersistentFlags().String(flagLogLevel, zerolog.InfoLevel.String(), "logging level")
 	rootCmd.PersistentFlags().String(flagLogFormat, logLevelText, "logging format; must be either json or text")
 	rootCmd.PersistentFlags().Bool(flagSkipProviderCheck, false, "skip the coingecko API provider check")
+	rootCmd.PersistentFlags().Bool(
+		flagChainConfig,
+		false,
```

Comment:
What do you think about making it on by default? If you want to override the on chain values you have to add the flag.

Reply:
I think its better actually yeah. I reversed it to be called `config-currency-providers` which when true uses the config file and is false by default.